playground/src/screens/TooltipsScreen.tsx

```diff
@@ -0,0 +1,167 @@
+import { NavigationComponent, NavigationComponentProps, Options } from 'react-native-navigation';
+import Navigation from '../services/Navigation';
+import React from 'react';
+
+import Root from '../components/Root';
+import Button from '../components/Button';
+import Screens from './Screens';
+import testIDs from '../testIDs';
+import { stack } from '../commons/Layouts';
+interface Props extends NavigationComponentProps {
+  enablePushBottomTabs: boolean;
+}
+export default class TooltipsScreen extends NavigationComponent<Props> {
+  public static defaultProps = {
+    enablePushBottomTabs: true,
+  };
+
+  static options(): Options {
+    return {
+      topBar: {
+        title: {
+          text: 'Tooltips screen',
+        },
+        rightButtons: [
+          {
+            text: 'Hit',
+            id: 'HitRightButton',
+          },
+        ],
+        backButton: {
+          testID: testIDs.BACK_BUTTON,
+        },
+      },
+      layout: {
+        orientation: ['portrait', 'landscape'],
+      },
+    };
+  }
+
+  constructor(props: Props) {
+    super(props);
+    Navigation.events().bindComponent(this);
+  }
+
+  render() {
+    return (
+      <Root componentId={this.props.componentId}>
+        <Button
+          label="showToolTips on Main BottomTabs Hit TopBar Button"
```

Comment:
all labels here are not fully visible in the app, let's change it to something shorter, for example, "Show on BottomTabs, Attached to Button"